src/bpftrace.cpp

```diff
@@ -1024,12 +1024,27 @@ int BPFtrace::print_map(IMap &map, uint32_t top, uint32_t div)
     old_key = key;
   }
 
-  if (map.type_.type == Type::count || map.type_.type == Type::sum || map.type_.type == Type::integer)
+  if (map.type_.type == Type::count || map.type_.type == Type::sum ||
+      map.type_.type == Type::integer)
```

Comment:
Ditto about code style changes. (I thought something actually changed in this line, but it's the same if)